Pages/HomePage/homepage.css

```diff
@@ -56,7 +58,7 @@
 }
 .searchIcon{
     font-size: 20px ;
```

Comment:
avoid giving font-size in pixels  `font-size: 20px ` replace `font-size: 1.25rem ` (20px equals to 1.25rem )